docs/share.md

```diff
@@ -3,6 +3,85 @@ id: share
 title: Share
 ---
 
+## Example
+
+<div class="toggler">
+  <ul role="tablist" class="toggle-syntax">
+    <li id="functional" class="button-functional" aria-selected="false" role="tab" tabindex="0" aria-controls="functionaltab" onclick="displayTabs('syntax', 'functional')">
+      Function Component Example
+    </li>
+    <li id="classical" class="button-classical" aria-selected="false" role="tab" tabindex="0" aria-controls="classicaltab" onclick="displayTabs('syntax', 'classical')">
+      Class Component Example
+    </li>
+  </ul>
+</div>
+
+<block class="functional syntax" />
+
+```SnackPlayer name=Function%20Component%20Example
+import React, { Component } from "react";
+import { Share, Button } from "react-native";
+const ShareExample = () => {
```

Comment:
I think that `export default ShareExample ...` will be better for the consistency between examples.